packages/utilities/gel-foundations/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 5.0.0 | [PR#3805](https://github.com/bbc/psammead/pull/3805) BBC Reith Qalam v1.100 |
```

Comment:
As this is a breaking change, should there be anything added to the readme telling someone how to migrate the next version? Am I right in thinking they should just use `arabic` rather `arabicPashto` for pashto pages?

Reply:
I have updated the PR description on https://github.com/bbc/psammead/pull/3805 - does that suffice, or should I put it in the README too? e.g. Removal of `arabicPashto` script - `arabic` should be used instead?

Reply:
In reality, no one else uses this I would assume, if people did it would be good to see it in the readme. Probably the PR description suffices :)